### PR TITLE
Add tests for EventsController methods

### LINKED ISSUE #4736: [GreenCity] - Create tests for EventsController

### PR DESCRIPTION
**Pull request by Maksym-Lenets**
Add tests for EventsController methods getUserEvents, addAttender, removeAttender, rateEvent, getAllEventSubscribers

### CODE REVIEW

**Review by NKorzhik — APPROVED** [review]

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 1]
```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
...
+    @SneakyThrows
+    void getUserEventsTest() {
+        int pageNumber = 0;
+        int pageSize = 20;
+
+        Pageable pageable = PageRequest.of(pageNumber, pageSize);
+        mockMvc.perform(get(EVENTS_CONTROLLER_LINK + "/myEvents").principal(principal))
+            .andExpect(status().isOk());
+
+        verify(eventService, times(1))
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java]: `times(1)` could be removed
  `verify(eventService)` equals to   `verify(eventService, times(1))`

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 2]
```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
...
+            .getAllUserEvents(
+                eq(pageable),
+                eq("test@gmail.com"));
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java]: please check if you call `getAllUserEvents` with `principal` email

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 3]
```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
...
+    @SneakyThrows
+    void addAttenderTest() {
+        Long eventId = 1L;
+
+        mockMvc.perform(post(EVENTS_CONTROLLER_LINK + "/addAttender/{eventId}", eventId).principal(principal));
+
+        verify(eventService, times(1))
+            .addAttender(
+                eq(eventId),
+                eq("test@gmail.com"));
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java]: same here

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 4]
```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
...
+            .removeAttender(
+                eq(eventId),
+                eq("test@gmail.com"));
+    }
+
+    @Test
+    @SneakyThrows
+    void rateEventTest() {
+        Long eventId = 1L;
+        int grade = 1;
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java]: please use different values for params

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 5]
```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
...
+                eq(grade));
+    }
+
+    @Test
+    @SneakyThrows
+    void getAllEventSubscribersTest() {
+        Long eventId = 1L;
+
+        mockMvc.perform(get(EVENTS_CONTROLLER_LINK + "/getAllSubscribers/{eventId}", eventId))
+            .andExpect(status().isOk());
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java:440]: You expect  `andExpect(status().isOk())` for every test
Could you have another response codes?
If yes then please add tests for that cases too

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 6]
```diff
@@ -22,8 +22,10 @@
 import java.security.Principal;
 
 import static greencity.ModelUtils.getPrincipal;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java]: please do not use `*`

File: core/src/test/java/greencity/controller/EventsControllerTest.java [hunk 7]
```diff
@@ -67,4 +69,76 @@ void getEventsCreatedByUser() {
 
     }
 
+    @Test
+    @SneakyThrows
+    void getUserEventsTest() {
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/EventsControllerTest.java:102]: Could you please mock `eventService` response and check the response

**Review by sergoliarnik — APPROVED** [review]

**Review by RomaMocherniuk — APPROVED** [review]

**Review by JJJazl — APPROVED** [review]

**Pull request merged**